packages/p2p/package-lock.json

```diff
@@ -0,0 +1,14473 @@
+{
```

Comment:
In monoRepo we should not have package-lock in each file
Please remove this package.lock and run `npm run bootstrap:dev` in the root to regenerate the package.lock with the new changes

Reply:
ah sorry accidentally added this, will remove 🙏